Src/DeUrgenta.Admin.Api/Commands/UpdateEvent.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using CSharpFunctionalExtensions;
+using DeUrgenta.Admin.Api.Models;
+using DeUrgenta.Common.Models;
+using MediatR;
+
+namespace DeUrgenta.Admin.Api.Commands
+{
+    public class UpdateEvent : IRequest<Result<EventModel>>
```

Comment:
Add / Update Models should have FluentValidation other them

Reply:
I was under impression this is covered by the EventRequestValidator (validates EventRequest parameter of the Add/Update operations)

Reply:
I think I misplaced the comment.
`EventRequest` should have a FluentValidator

FluentValidators enforce model rules (string lengths, min /max dates)
EventRequestValidator (and other validators which are used in Handlers) enforce business rules.